lib/core/src/Cardano/Wallet.hs

```diff
@@ -2118,6 +2127,8 @@ data FeeEstimation = FeeEstimation
     -- ^ Most coin selections will result in a fee higher than this.
     , estMaxFee :: Word64
     -- ^ Most coin selections will result in a fee lower than this.
+    , estDeposit :: Word64
+    -- ^ Deposit if stake key was registered
```

Comment:
This data-type feels a little bit _abused_. There's nothing of an estimation when it comes to stake key deposits. The deposit value is fixed by the network.